publication-rest/src/test/java/no/unit/nva/publication/update/UpdatePublicationHandlerTest.java

```diff
@@ -999,7 +1000,8 @@ void shouldUpdateFilesForApprovalWhenPublicationUpdateHasFileChanges() throws Ap
     }
 
     @Test
-    void curatorShouldBeAbleToOverrideRrs() throws IOException, NotFoundException {
+    void publishingCuratorWithAccessRightManageResourceFilesShouldBeAbleToOverrideRrs()
```

Comment:
I assume that publishing curator only should have access to override rrs, since rrs is a part of file metadata